index.html

```diff
@@ -1,155 +1,47 @@
-    <tr>
-        <td>&lt; tbody &gt;</td>
-        <td>Groups the body content in a table</td>
-    </tr>
-    <tr>
-        <td>&lt; tfoot &gt;</td>
-        <td>Groups the footer content in a table</td>
-    </tr>
-    </tbody>
-</table>
-<br>
-
-<form action="/" method="GET">
-    <fieldset>
-        <legend>My first Form</legend>
-        <div>
-            <label for="name">Name</label><br>
-            <input id="name" type="text" name="name" placeholder="Name" required>
-        </div>
-        <div>
-            <label for="email">Email</label><br>
-            <input id="email" type="email" name="email" placeholder="ivanov@gmail.com" required>
-        </div>
-        <div>
-            <label for="pass">Password</label><br>
-            <input id="pass" type="password" name="password" required>
-        </div>
-        <br>
-        <div>
-            <label for="range">Your programming level</label><br>
-            <input id="range" type="range" name="level" min="0" max="10">
-        </div>
-        <div>
-            <p>What does HTML stand for?</p>
-            <input id="1" type="checkbox" name="answer_html" value="false">
-            <label for="1">Hyperlinks and text Markup Language</label><br>
-            <input id="2" type="checkbox" name="answer_html" value="false">
-            <label for="2">Home Tool Markup Language</label><br>
-            <input id="3" type="checkbox" name="answer_html" value="true" checked>
-            <label for="3">Hyper Text Markup Language</label>
-        </div>
-        <div>
-            <p>What does HTTP stand for?</p>
-            <input id="radio_1" type="radio" name="answer_http" value="false">
-            <label for="radio_1">Hyper Text Translate Protocol</label><br>
-            <input id="radio_2" type="radio" name="answer_http" value="false">
-            <label for="radio_2">Hyper Text Transmission Protocol</label><br>
-            <input id="radio_3" type="radio" name="answer_http" value="true" checked>
-            <label for="radio_3">Hyper Text Transfer Protocol</label>
-        </div>
-        <br>
-        <div>
-            <label for="comment">Comment:</label><br>
-            <textarea id="comment" name="comment" cols="30" rows="5" maxlength="500">Value</textarea>
-        </div>
-        <br>
-        <div>
-            <label for="select">Which attributes does select tag have?</label><br><br>
-            <select id="select" name="select_attr">
-                <option value="" selected>Your answer</option>
-                <option value="option_true">option</option>
-                <option value="color">color</option>
-                <option value="multiple">display</option>
-            </select>
-        </div>
-        <br>
-        <input type="submit" value="Send">
-    </fieldset>
-</form>
-
+<div class="offer-container">
+    <h3 class="offer-title">What do we offer</h3>
+    <div class="offer-title-line"></div>
+    <div class="offer-blocks-wrapper">
```

Comment:
**Semantic recommendation** 
It's better to use `<ul>`  with` <li>` here

Reply:
Replaced div for ul/li